src/features/marking-coordinate/coordinate-info.tsx

```diff
@@ -47,18 +47,20 @@ const CoordinateGroup = styled.div`
 `
 
 export const CoordinateInfo = () => {
-  const { endPos, startPos, cursorPos } = useMarkCoordinate()
+  const { endPos, polygons, startPos, cursorPos } = useMarkCoordinate()
   const { status } = useStatusUpdate()
 
   const { pollingJobId, pollingError, pollingStatus } = useViewDamage()
 
   return (
+    // TODO: show the polygons, somehow ... but not like this:
```

Comment:
@RitwikGupta there are 5 TODOs in this PR, all of which could/should probably be addressed somehow ... but not this evening, at least, not by me :)